lib/tuya.js

```diff
@@ -1656,15 +1656,15 @@ const tuyaTz = {
             'scale_protection', 'error', 'radar_scene', 'radar_sensitivity', 'tumble_alarm_time', 'tumble_switch', 'fall_sensitivity',
             'min_temperature', 'max_temperature', 'window_detection', 'boost_heating', 'alarm_ringtone', 'alarm_time', 'fan_speed',
             'reverse_direction', 'border', 'click_control', 'motor_direction', 'opening_mode', 'factory_reset', 'set_upper_limit', 'set_bottom_limit',
-            'motor_speed',
+            'motor_speed', 'backlight_mode', 'calibration', 'motor_steering',
         ],
         convertSet: async (entity, key, value, meta) => {
             // A set converter is only called once; therefore we need to loop
             const state = {};
             const datapoints = utils.getMetaValue(entity, meta.mapped, 'tuyaDatapoints', undefined, undefined);
             if (!datapoints) throw new Error('No datapoints map defined');
             for (const [key, value] of Object.entries(meta.message)) {
-                const convertedKey = meta.mapped.meta.multiEndpoint ? `${key}_${meta.endpoint_name}` : key;
+                const convertedKey = meta.mapped.meta.multiEndpoint && (meta.endpoint_name || false) ? `${key}_${meta.endpoint_name}` : key;
```

Comment:
I see why you changed this now, maybe a small improvement, does `meta.mapped.meta.multiEndpoint && meta.endpoint_name ? `${key}_${meta.endpoint_name}` : key;` also work?

Reply:
should work, will fix it